src/NServiceBus.AcceptanceTests/Core/TestSuiteConstraints.cs

```diff
@@ -8,7 +8,7 @@ public partial class TestSuiteConstraints
 
         public bool SupportsCrossQueueTransactions => true;
 
-        public bool SupportsNativePubSub => true;
+        public bool SupportsNativePubSub => false;
```

Comment:
Should this explain why we set it this way? I think anyone would probably switch it back if no explanation is given

Reply:
good point 👍

Reply:
@danielmarbach I added a comment, can you review and approve if you're happy with it?